File: DKImagePickerControllerDemo_OC/DKImagePickerControllerDemo_OC/CustomLayoutUIDelegate/CustomLayoutUIDelegate.m
//
//  CustomLayoutUIDelegate.m
//  DKImagePickerControllerDemo_OC
//
//  Created by 赵铭 on 2017/7/10.
//  Copyright © 2017年 zm. All rights reserved.
//

#import "CustomLayoutUIDelegate.h"
#import "CustomFlowLayout.h"
@implementation CustomLayoutUIDelegate
- (UICollectionViewLayout *)layoutForImagePickerController:(DKImagePickerController *)imagePickerController{
    return [CustomFlowLayout new];
}


@end

File: Example/DKImagePickerControllerDemo/CustomLayoutUIDelegate/CustomLayoutUIDelegate.swift
//
//  CustomLayoutUIDelegate.swift
//  DKImagePickerControllerDemo
//
//  Created by ZhangAo on 03/01/2017.
//  Copyright © 2017 ZhangAo. All rights reserved.
//

import UIKit
import DKImagePickerController

open class CustomLayoutUIDelegate: DKImagePickerControllerBaseUIDelegate {
    
    override open func layoutForImagePickerController(_ imagePickerController: DKImagePickerController) -> UICollectionViewLayout.Type {
        return CustomFlowLayout.self
    }
    
}
